FlixHqMovie/FlixHqMovie/Sences/HomeScreen/HeaderTableView/HeaderTableView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  HeaderTableView.swift
+//  MVVM_Swift
+//
+//  Created by DuyThai on 22/02/2023.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+final class HeaderTableView: UITableViewCell {
+    static let identifier = "HeaderTableView"
+    @IBOutlet private weak var headerTableImageView: UIImageView!
+
+    @IBOutlet private weak var addMyListButton: UIButton!
+    @IBOutlet private weak var playButton: UIButton!
+    var onPlayButtonTapped: ((String) -> Void)?
+    let disposeBag = DisposeBag()
+    private var id = ""
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        playButton.rx.tap.subscribe(onNext: {
+            self.onPlayButtonTapped?(self.id)
+        }).disposed(by: disposeBag)
```

Comment:
break line rồi format nó lại e
```suggestion
        playButton.rx.tap
        .subscribe(onNext: {
            self.onPlayButtonTapped?(self.id)
        })
        .disposed(by: disposeBag)
```